src/components/Company/Offers/Manage/CompanyOffersManagementWidget.js

```diff
@@ -50,15 +51,16 @@ const RowActions = () => (
 
 
 const CompanyOffersManagementWidget = () => {
+    const { data } = useSession();
     const [offers, setOffers] = useState(null);
     const [isLoading, setIsLoading] = useState(true);
     const [error, setError] = useState(null);
 
     useEffect(() => {
-        fetchCompanyOffers().then((offers) => {
+        fetchCompanyOffers(data.company._id).then((offers) => {
```

Comment:
check if `isLoggedIn` before executing

```suggestion
        if(isLoggedIn) fetchCompanyOffers(data.company._id).then((offers) => {
```